examples/stable_baselines3/run.py

```diff
@@ -0,0 +1,204 @@
+import gym
+import os
+import pathlib
+import numpy as np
+import tensorflow as tf
+from shutil import copyfile
+from ruamel.yaml import YAML
+
+from smarts.core import agent as smarts_agent
+from smarts.core import agent_interface as smarts_agent_interface
+from smarts.core import controllers as smarts_controllers
+from smarts.env import hiway_env as smarts_hiway_env
+import env.rgb_image as smarts_rgb_image
+import env.single_agent as smarts_single_agent
+import env.reward as reward
+import env.action as action
+
+from stable_baselines3 import PPO
+from stable_baselines3.common import monitor
+from stable_baselines3.common.env_checker import check_env
+from stable_baselines3.common.evaluation import evaluate_policy
+from argument_parser import default_argument_parser
+
+
+yaml = YAML(typ="safe")
+
+
+def create_env(config):
```

Comment:
Don't mix environment creation with training/evaluation script. Move environment creation `create_env()` function to a separate file.